core/imt/test/testEnableImt.cxx

```diff
@@ -0,0 +1,21 @@
+#include "TROOT.h"
+#include "ROOT/RTaskArena.hxx"
+
+#include "ROOT/TestSupport.hxx"
+#include "gtest/gtest.h"
+
+#ifdef R__USE_IMT
+
+static const unsigned gMaxConcurrency = ROOT::Internal::LogicalCPUBandwidthControl();
+
+TEST(EnableImt, TBBAttach)
+{
+   ROOT::EnableImplicitMT(ROOT::EIMTConfig::kExistingTBBArena);
+
+   auto psize = ROOT::GetThreadPoolSize();
+
+   EXPECT_TRUE( psize > 1);
+   EXPECT_EQ( psize, gMaxConcurrency);
```

Comment:
@Dr15Jones Is there an 'easy' way to test that the new call those what we are hoping for?

Reply:
You could have the external arena declare only 1 level of concurrency and then see if that is what you get back.